generated/DictionaryMetaSchema.json

```diff
@@ -38,10 +38,35 @@
     "Meta": {
       "type": "object",
       "additionalProperties": {
-        "type": [
-          "string",
-          "number",
-          "boolean"
+        "anyOf": [
+          {
+            "anyOf": [
+              {
+                "type": "string"
+              },
+              {
+                "type": "number"
+              },
+              {
+                "type": "boolean"
+              },
+              {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
+              },
+              {
+                "type": "array",
+                "items": {
+                  "type": "number"
```

Comment:
unclear why array of booleans are not needed/allowed.
 
worth commenting?

Reply:
this is a generated file. I can comment this point but not here.